README.md

```diff
@@ -269,6 +269,36 @@ cmake [-G <platform-generator>] <spirv-dir>
 Once the build files have been generated, build using your preferred
 development environment.
 
+### Tools you'll need
+
+For building and testing SPIRV-Tools, the following tools should be
+installed regardless of your OS:
+
+- [CMake](http://www.cmake.org/): for generating compilation targets.  Version
+  2.8.12 or later.
+- [Python](http://www.python.org/): for utility scripts and running the test 
+suite. Version 2 or 3.
```

Comment:
So, if we are saying Python 2 or 3, should the bots make sure that they test with both?

Reply:
I changed the windows bots so they use python3.  Is that enough?

Reply:
But the Linux bot is also using python3.  I was thinking that maybe we leave one bot running python2.

Reply:
Look at https://source.cloud.google.com/results/invocations/440d865b-cb1d-4da4-8581-133a2ad7cb62/targets/graphics_shader_compiler%2Fspirv-tools%2Flinux-clang-debug%2Fpresubmit/log

You will find 

> -- SPIRV-Tools: nosetests was not found - python support code will not be tested
> -- Found PythonInterp: /home/kbuilder/.pyenv/shims/python (found version "2.7.6")

Reply:
True, we did not force python2 for those builds.

Reply:
Done I made it explicity 2.7 for linux bots.